MANIFEST.in

```diff
@@ -0,0 +1 @@
+recursive-include schema/ *.json
```

Comment:
I think maybe we want to include `recursive-include examples/ *.json` as well, else one would not be able to test locally.

Reply:
Pushed